composer.json

```diff
@@ -24,7 +24,7 @@
         "issues": "https://github.com/RusticiSoftware/TinCanPHP/issues"
     },
     "require": {
-        "php": "~5.4",
+        "php": "~5.5 || ^7.0",
```

Comment:
@xabbuh yes, as the PHP core team has dropped support for it. Ref: http://php.net/supported-versions.php